test/entrypoint/srv_select/storage_init.lua

```diff
@@ -229,6 +230,102 @@ return function()
         if_not_exists = true,
     })
 
+    if decimal_supported then
+        local decimal_format = {
+            {name = 'id', type = 'unsigned'},
+            {name = 'bucket_id', type = 'unsigned'},
+            {name = 'decimal_field', type = 'decimal'},
+        }
+
+
```

Comment:
It was intentional since there is a hundred lines of code and it would be nice to visually separate one space setup from the other. But I can remove them, if you insist.